templates/legal/data-privacy/index.html

```diff
@@ -272,6 +272,35 @@ <h3>What is a cookie?</h3>
               <p><a href="https://salesloft.com/privacy-notice/">Salesloft's privacy notice</a></p>
             </td>
           </tr>
+          <tr>
+            <td>Reddit</td>
+            <td><code>_rdt_uuid</code>, <code>reddaid</code></td>
+            <td>
+              <p>Using the Reddit Pixel, we can build a retargeting audience. This helps us reconnect with people who’ve shown interest.</p>
+              <p><a href="https://www.reddit.com/policies/privacy-policy">Reddit's privacy policy</a></p>
+            </td>
+          </tr>
+          <tr>
+            <td>ZoomInfo</td>
+            <td><code>ab</code>, <code>u</code>, <code>ssi</code>, <code>demdex</code>, <code>dpm</code>, <code>_cc_id</code>,
+              <code>_cc_dc_</code>, <code>rla3</code>, <code>pxrc</code>,
+              <code>IDE</code>, <code>i</code>, <code>CookieConsent</code>, <code>__cf_bm</code>, 
+              <code>lrsync</code>, <code>_ga</code>, <code>chs</code>, <code>visitorid</code>, <code>insent-user-id</code>,
+              <code>_zitok</code>, <code> _px3</code>, <code>__cf_bm</code>
```

Comment:
There seems to be an additional space here:
```suggestion
              <code>_zitok</code>, <code>_px3</code>, <code>__cf_bm</code>
```